_posts/2025-04-23-secure-mcp-sse-server.adoc

```diff
@@ -0,0 +1,202 @@
+---
+layout: post
+title: 'Getting ready for secure MCP with Quarkus MCP Server'
+date: 2025-04-23
+tags: ai mcp security
+synopsis: 'Demonstrate secure Quarkus MCP SSE server access in the dev mode'
+author: sberyozkin
+---
+:imagesdir: /assets/images/posts/secure_mcp_sse_server
+
+== Introduction
+
+https://modelcontextprotocol.io/specification/2025-03-26[The latest version of the Model Context Protocol (MCP) specification] introduces an https://modelcontextprotocol.io/specification/2025-03-26/basic/authorization[authorization] flow for HTTP based MCP transports.
+
+MCP authorization will enable MCP SSE and Streamable HTTP clients to access MCP servers securely and is being very actively discussed and evaluated right now. This https://github.com/modelcontextprotocol/modelcontextprotocol/pull/284[MCP authorization PR] is one of the focal points for the MCP authorization discussions.
```

Comment:
I dont understand the first sentence. First, I would remove securely - as it is only about authentication.